docs/topics/development/debugging.rst

```diff
@@ -90,6 +88,14 @@ If clicked, it looks like:
 
 .. image:: /screenshots/django-debug-toolbar-expanded.png
 
+You must know that using the Django Debug Toolbar will slow the website quite a
+lot. You can mitigate this by deselecting the checkbox next to the ``SQL``
+panel.
+
+Also, please note that you should only use the Django Debug Toolbar if you need
+it, as it disables CSP for your local dev. CSP will still report any issues in
+the console, allowing you to fix them.
+
```

Comment:
@muffinresearch added the above paragraph on CSP, what do you think?
@andymckay yeah, I'd rather not make it too complicated for a contributor to benefit from the DjDT (and at the same time put them at risk on the whole internet ;)